### PR TITLE
Run initial decom of legacy ELK

### PR DESCRIPTION
**Pull request by cloudnull**
These commits remove the legacy ELK tooling from our stack and document the new deployment process when manual steps are used.

Issue: [FLEEK-82](https://rpc-openstack.atlassian.net/browse/FLEEK-82)

Issue: [FLEEK-130](https://rpc-openstack.atlassian.net/browse/FLEEK-130)

### CODE REVIEW

**Comment by prometheanfire** [comment]
recheck_xenial_no_artifacts_swift_deploy

File: playbooks/vars/openstack-service-config.yml [hunk 1]
```diff
@@ -72,7 +72,6 @@ openstack_images:
       hw_disk_bus: "scsi"
       hw_vif_multiqueue_enabled: "true"
       hw_qemu_guest_agent: "yes"
-      hypervisor_type: "qemu"
```
Comment (prometheanfire) [inline playbooks/vars/openstack-service-config.yml]: why was this taken out?  iirc we need it to ensure nova doesn't try to schedule these images to ironic nodes...

Comment (cloudnull) [reply playbooks/vars/openstack-service-config.yml]: in master / queens qemu != kvm

Comment (prometheanfire) [reply playbooks/vars/openstack-service-config.yml]: ok, so should we change this to kvm?

Comment (cloudnull) [reply playbooks/vars/openstack-service-config.yml]: done

**Comment by prometheanfire** [comment]
wat

15:54:30 EXIT NOTICE [Playbook execution success] **************************************

**Comment by cjloader** [comment]
recheck_xenial_no_artifacts_swift_deploy

**Comment by cjloader** [comment]
fatal: [localhost]: UNREACHABLE! => {"changed": false, "msg": "Failed to connect to the host via ssh: Host key verification failed.\r\n", "unreachable": true}

recheck_xenial_no_artifacts_swift_deploy

**Comment by odyssey4me** [comment]
recheck_container_ansible-syntax_tox-test

**Pull request merged**